daffodil-test-ibm1/src/test/scala-debug/org/apache/daffodil/IBMTests2.scala

```diff
@@ -44,10 +44,8 @@ class IBMTestsThatThrow {
   lazy val runner1 = new DFDLTestSuite(Misc.getRequiredResource(tdml1))
   lazy val runner2 = new DFDLTestSuite(Misc.getRequiredResource(tdml2))
 
-  @Test def test_scoping_define_format_8_04() { runner1.runOneTest("scoping_define_format_8_04") } //DFDL-565 - attributeFormDefault='qualified'
-
-  @Test def test_scoping_default_format_8_01() { runner1.runOneTest("scoping_default_format_8_01") } // DFDL-565 attributeFormDefault='qualified'
-  @Test def test_scoping_define_format_8_01() { runner1.runOneTest("scoping_define_format_8_01") } // DFDL-565 attributeFormDefault='qualified'
+  @Test def test_scoping_default_format_8_01() { runner1.runOneTest("scoping_default_format_8_01") } // Attribute "encodingErrorPolicy" was already specified - bug in IBM test
+  @Test def test_scoping_define_format_8_01() { runner1.runOneTest("scoping_define_format_8_01") } // Attribute "encodingErrorPolicy" was already specified - bug in IBM test
```

Comment:
Does this test pass if the duplicate errorEncodingPolicy is removed? These tests are more about how properties are resolved, and not really about how duplicate properties are handled (which I agree, seems likea bug that IBM DFDL should catch). I'd guess the duplicate property was unintentional and can be removed and still maintain the intention of the test.

Reply:
This feels like our mistake adding the duplicate encodingErrorPolicy property. That is, this property encodingErrorPolicy, is relatively new. Probably hadn't even been invented when we first got a bunch of preliminary DFDL tests from IBM for incorporation into Daffodil years ago. So we subsequently added this to keep the tests working, once the new property was added to Daffodil, and required. I think adding it twice was just a mistake we made.

Reply:
They also need textOutputMinLength added. I wasn't sure if we wanted to change these tests since they were from IBM. Is it ok to update them?

Reply:
[DAFFODIL-1616](https://issues.apache.org/jira/browse/DAFFODIL-1616) is the bug to fix the issue related to textOutputMinLength, which Mike is working on. If the tests work with that property added, it seems fine to me to modify the tests and add the property, and eventually that property will just be ignored when Mike's fix is in.